scripts/apps/search/directives/SearchFilters.ts

```diff
@@ -40,7 +41,10 @@ class LinkFunction {
         // fetch available languages
         metadata.initialize()
             .then(() => {
-                scope.category = metadata?.cvs?.find((cv) => cv._id === 'categories');
+                this.scope.filterLabels = {};
+                metadata?.cvs?.forEach((cv) => {
+                    this.scope.filterLabels[cv._id] = gettext(cv.display_name);
```

Comment:
`gettext` won't work on dynamic user generated and editable values like vocabulary names. Vocabulary names aren't translatable yet. You should add an additional object if you want to handle translations.

I think it'd work to add translations object at the root of the vocabulary which would support multiple fields:
![screenshot-2020-09-23--13-18](https://user-images.githubusercontent.com/2076953/94005935-596a7d80-fd9f-11ea-8d43-5beec0faea9b.png)

What do you think @petrjasek ?

Reply:
yep that would work